pritunl/sso/azure.py

```diff
@@ -183,7 +183,7 @@ def _verify_azure_2(user_name):
                 'Content-Type': 'application/json',
             },
             json={
-                'securityEnabledOnly': 'false',
+                'securityEnabledOnly': str(settings.app.sso_azure_security_groups_only).lower(),
```

Comment:
Having the option to sync all groups or only security groups. Default will still be false, which will collect all groups for the user and not only security groups.